homeassistant/components/tuya/light.py

```diff
@@ -257,6 +257,14 @@ class ColorTypeData:
     v_type=IntegerTypeData(min=1, scale=0, max=255, step=1),
 )
 
+# Used for devices with "colour_data" key and 0-1000 range
+# Strip lights use 0-1000 but this is not documented in Tuya API docs
+DEFAULT_COLOR_TYPE_DATA_1000 = ColorTypeData(
```

Comment:
This is a duplicate of `DEFAULT_COLOR_TYPE_DATA_V2`

Reply:
Yes its a duplicate since the range is the same.
My thought was to separate it from DEFAULT_COLOR_TYPE_DATA_V2 to make it clear "colour_data_v2" key is not used for API call but "colour_data"

Just tested with this code and it seems to work fine:
# If no type is found, use a default one
self._color_data_type = DEFAULT_COLOR_TYPE_DATA
if self._color_data_dpcode == DPCode.COLOUR_DATA_V2 or (
    self._brightness_type and self._brightness_type.max > 255
) **_or (self.device.category == "dd")_**:
    self._color_data_type = DEFAULT_COLOR_TYPE_DATA_V2
 Then we can delete the DEFAULT_COLOR_TYPE_DATA_1000

Reply:
> Then we can delete the DEFAULT_COLOR_TYPE_DATA_1000

👍